src/toil_lib/tools/__init__.py

```diff
@@ -2,6 +2,17 @@
 import subprocess
 
 
+def log_runtime(job, start, end, cmd):
```

Comment:
Given the repetitiveness of having to call `time.time()` before and after every dockerCall along with this function, would this be better refactored as a decorator for `dockerCall`?